test/functional/services/visualizations/inspector.js

```diff
@@ -0,0 +1,149 @@
+/*
```

Comment:
Should this service really be nested under `visualizations`? Inspector is used in Discover, Dashboard, and Maps plugin. All of which are not visualizations.

Reply:
"visualizations" term is overloaded.  I was more for like "anything that can be visualized, including saved searches, maps, etc".  A map is a "visualization" of a type.  Maybe "embeddable" is more appropriate in this context, but these things might not actually be embeddables either.  

Would you put it top level?  I suppose I can do that for now, only one file under this folder anyway, and it's confusing.

Reply:
actually I take that back, "pie chart" is in here, but I can still move to top level.